testsuite/cluster-test/src/cluster.rs

```diff
@@ -228,4 +228,34 @@ impl Cluster {
         self.all_instances()
             .filter(move |v| v.validator_group() == validator_group)
     }
+
+    pub fn get_lsr_instances (
```

Comment:
Cluster has `lsr_instances` method, that returns lsr instances on the cluster, so `get_lsr_instances` name is confusing. Should we rename this to something like `lsr_instances_for_validators`?